### DISCUSSION TITLE
multiple swipers in single page

### DISCUSSION THREAD

**Discussion by neobie** [nolimits4web/swiper#5649] [Q&A]
### Clear and concise description of the problem

https://stackoverflow.com/questions/32118999/how-can-i-have-multiple-swiper-slideshows-on-a-single-page

I wonder in 2022, initializing multiple swipers in single page still not working ?
Do we still need to initialize 1 by 1 with different instances?

### Suggested solution

We need $(".swiper").init()

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### Would you like to open a PR for this feature?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
Is it still a problem to use loops in 2022? 
```
$('.swiper').each((el) => {
  new Swiper(el, ...)
})
```

**Reply by neobie** [reply]
Thanks. Latest version did that. Awesome and sorry for trouble.

**Reply by cfxd** [reply]
@nolimits4web Just wondering, is there any advantage/disadvantage to keeping a master array of all Swipers on a page? For example:

```js
const swipers = [];
document.querySelectorAll('.swiper').forEach(function(el) {
  const swiper = new Swiper(el, {
    loop: true,
    loopPreventsSlide: false,
    navigation: {
      nextEl: '.swiper-button-next',
      prevEl: '.swiper-button-prev',
    },
  });
  swipers.push(swiper);
});
```
